benchexec/tools/sv_benchmarks_util.py

```diff
@@ -0,0 +1,28 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+This module contains some useful functions related to tasks in the sv-benchmarks
+repository: https://github.com/sosy-lab/sv-benchmarks
+"""
+
+import benchexec.tools.template
+
+def get_data_model_from_task(task, data_models):
```

Comment:
Please add documentation. Shouldn't `data_models` be renamed because it does not primarily contain the data models, but command-line parameters (or values) for data models?

Reply:
fixed in 6f11a4cb9c66b01effc2b0a591ce95e368f99631